Pod/Classes/MenuView.swift

```diff
@@ -8,6 +8,16 @@
 
 import UIKit
 
+public struct MultilineMenuItem: MenuItemType {
```

Comment:
oh I don't have any plan to provide view only for now.
I forgot to mention that.